addons/purchase_stock/models/stock.py

```diff
@@ -257,6 +257,30 @@ def _set_default_route_id(self):
             orderpoint_wh_supplier.route_id = route_id[0].id
         return super()._set_default_route_id()
 
+    def action_replenish(self):
```

Comment:
Move this method near `action_view_purchase` to keep this structure:
1. Private attributes (_name, _description, _inherit, …)
2. Default method and _default_getField declarations
3. Field declarations
4. **Compute, inverse and search methods in the same order as field declaration**
5. Selection method (methods used to return computed values for selection fields)
6. Constrains methods (@api.constrains) and onchange methods (@api.onchange)
7. CRUD methods (ORM overrides)
8. **Action methods**
9. **And finally, other business methods.**